internal/armada/repository/apimessages/conversions.go

```diff
@@ -239,15 +239,28 @@ func FromInternalJobSucceeded(queueName string, jobSetName string, time time.Tim
 	if err != nil {
 		return nil, err
 	}
+
+	apiEvent := &api.JobSucceededEvent{
```

Comment:
Wouldn't it be better to have a function in eventutil? Like this but for log sequence to api sequence?
https://github.com/G-Research/armada/blob/master/internal/common/eventutil/eventutil.go#L519

Reply:
I think it's a historical accident that the conversions from logEvent -> ApiEvent are here, while the reverse are in `EventUtil`.  That said I'm not convinced it's worth moving everything.  I hope the `EventUtil` ones will go away soon, while these ones are probably in the right place because the old `api.Event` should purely be confined to the serving of events to users.